README.md

```diff
@@ -29,31 +29,59 @@ Parameter|Value|Description
 #### Optional workflow parameters:
```

Comment:
Should `calculateTumorAwareResults` be listed in the parameters?

Reply:
Ugh, getting close to making wdl-tools a git hook. This did make me realize this needed to be a root parameter, as being able to change it for the different logical branches makes no sense.